canonical-websites/www.ubuntu.com
# Enterprise menu update

## Done
Updated the content of the enterprise menu to match the copy doc. Added source-maps to help find where styles are. Set the display to block on small screens to fix an FF quirk,

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Click the enterprise menu
- Check that it matches the [copy doc](https://docs.google.com/document/d/1YBdQvLuqEpEQr_QqyycxMhZr4OaLapMlNJyYKCmPJsQ/edit#)
- If you want, inspect an element and see where the styles come from.

## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/666
